docs/pull_request.md

```diff
@@ -0,0 +1,208 @@
+# Pull requests
+
+## Introduction
+
+This document gives some clue about how to efficiently manage Pull Requests (PR). This document is a first draft and may be improved later.
+
+## Who should read this document?
+
+Every pull request reviewers, but also probably every ones who submit PRs.
+
+## Submitting PR
+
+### Who can submit pull requests?
+
+Basically every one who wants to contribute to the project! But there are some rules to follow.
+
+#### Humans
+
+People with write access to the project can directly clone the project, push their branches and create PR.
+
+External contributors must first fork the project and create PR to the mainline from there.
+
+##### Draft PR?
+
+Draft PR can be created when the submitter does not expect the PR to be reviewed and merged yet. It can be useful to publicly show the work, or to do a self-review first.
+
+Draft PR can also be created when it depends on other un-merged PR.
+
+In any case, it is better to explicitly declare in the description why the PR is a draft PR.
+
+Also, draft PR should not stay indefinitely in this state. It may be removed if it is the case and the submitter does not update it after a few days.
+
+##### PR Review Assignment
```

Comment:
```suggestion
## PR Review Assignment
```

I think this section deserves higher visibility. It has a huge impact on daily work. We should be able to send people directly to this section and the title must be more visible.

Reply:
I agree, but it impacts the whole toc. I can update that later